Python/libraries/recognizers-text/recognizers_text/Matcher/simple_tokenizer.py

```diff
@@ -1,23 +1,70 @@
 from .ITokenizer import Tokenizer
+from .Token import Token
 
 
 class SimpleTokenizer(Tokenizer):
 
-    def tokenize(self, _input: str) -> []:
-        pass
+    def tokenize(self, input: str) -> []:
+        tokens = []
 
-    @staticmethod
-    def is_chinese(c: chr):
-        pass
+        if not input:
```

Comment:
I think you are testing if 'input' was set to None, right? In that case, check the [Programming Recommendations](https://www.python.org/dev/peps/pep-0008/#id51) doc. 

Basically, it recommends how to check these cases:

> Also, beware of writing "if x" when you really mean "if x is not None" -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!